components/search_results/index.tsx

```diff
@@ -69,12 +73,5 @@ function makeMapStateToProps() {
     };
 }
 
-function mapDispatchToProps(dispatch) {
-    return {
-        actions: bindActionCreators({
-            getMorePostsForSearch,
-        }, dispatch),
-    };
-}
-
-export default connect(makeMapStateToProps, mapDispatchToProps)(SearchResults);
+// eslint-disable-next-line @typescript-eslint/ban-types
+export default connect<StateProps, {}, OwnProps, GlobalState>(makeMapStateToProps)(injectIntl<'intl', Props>(SearchResults));
```

Comment:
Instead of injectIntl, we can use `useIntl` in SearchResults as well

Reply:
done